src/main/resources/zafira.properties

```diff
@@ -1,6 +1,6 @@
-zafira_enabled=false
+zafira_enabled=true
```

Comment:
please revert zafira.properties changes.
Those items will be provided on CI (jenkins level) in carina-demo project which is intergated into qps-infra